### PR TITLE
Basic Zookeeper and Kafka tasks added

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**
- Basic big data tasks added
- kafka PCA workflow updated

### CODE REVIEW

File: Zookeeper/METADATA.json [hunk 1]
```diff
@@ -27,6 +27,24 @@
...
+				"name" : "Zookeeper_Start_or_Bind_Service",
+				"metadata" : {
+					"kind": "workflow",
+					"commitMessage": "First commit",
+					"contentType": "application/xml"
+				},
+				"file" : "resources/catalog/Zookeeper_Start_or_Bind_Service.xml"
+			},
+			{
+				"name" : "Zokeeper_Terminate_Service",
```
Comment (MaelAudren) [inline Zookeeper/METADATA.json]: Zokeeper_Terminate_Service => Zookeeper_Terminate_Service

File: Zookeeper/METADATA.json [hunk 2]
```diff
@@ -27,6 +27,24 @@
...
+				"metadata" : {
+					"kind": "workflow",
+					"commitMessage": "First commit",
+					"contentType": "application/xml"
+				},
+				"file" : "resources/catalog/Zokeeper_Terminate_Service.xml"
```
Comment (MaelAudren) [inline Zookeeper/METADATA.json]: Zokeeper_Terminate_Service.xml => Zookeeper_Terminate_Service.xml

File: Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.9"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.9 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.9/schedulerjob.xsd"
+    name="Zokeeper_Terminate_Service" projectName="1. Zookeeper"
```
Comment (MaelAudren) [inline Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml]: Zokeeper_Terminate_Service => Zookeeper_Terminate_Service
\+ rename file

File: Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    <variable name="Number_of_tweets" value="2000" />
+  </variables>
+  <genericInformation>
+    <info name="bucketName" value="big-data"/>
+    <info name="pca.action.icon" value="https://s3.eu-west-2.amazonaws.com/activeeon-public/icons/zookeeper.jpg"/>
+    <info name="group" value="public-objects"/>
+    <info name="pca.action.icon" value="/cloud-automation/styles/patterns/img/twitter-r.png"/>
+  </genericInformation>
+  <taskFlow>
+    <task name="Zokeeper_Terminate_Service">
```
Comment (MaelAudren) [inline Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml]: Zokeeper_Terminate_Service => Zookeeper_Terminate_Service

File: Zookeeper/resources/catalog/Zookeeper_Start_or_Bind_Service.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+     maxNumberOfExecution="2">
+  <genericInformation>
+    <info name="bucketName" value="big-data"/>
+    <info name="pca.action.icon" value="https://s3.eu-west-2.amazonaws.com/activeeon-public/icons/zookeeper.jpg"/>
+    <info name="group" value="public-objects"/>
+  </genericInformation>
+  <taskFlow>
+    <task name="Zookeeper_Start_or_Bind_Service">
+      <description>
+        <![CDATA[ Start a Zokeeper instance, then add the instance name and endpoint to the list of variables ]]>
```
Comment (MaelAudren) [inline Zookeeper/resources/catalog/Zookeeper_Start_or_Bind_Service.xml]: Zokeeper => Zookeeper

File: Zookeeper/METADATA.json (hunk 2 above)
Comment (nebilbenmabrouk) [reply Zookeeper/METADATA.json]: done

File: Zookeeper/METADATA.json (hunk 1 above)
Comment (nebilbenmabrouk) [reply Zookeeper/METADATA.json]: done

File: Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml (hunk 3 above)
Comment (nebilbenmabrouk) [reply Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml]: done

File: Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml (hunk 4 above)
Comment (nebilbenmabrouk) [reply Zookeeper/resources/catalog/Zokeeper_Terminate_Service.xml]: done

File: Zookeeper/resources/catalog/Zookeeper_Start_or_Bind_Service.xml (hunk 5 above)
Comment (nebilbenmabrouk) [reply Zookeeper/resources/catalog/Zookeeper_Start_or_Bind_Service.xml]: done

**Pull request merged**